src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -694,32 +694,29 @@ private ISymbolNode GetHelperFtnUncached(CorInfoHelpFunc ftnNum)
                     break;
 
                 case CorInfoHelpFunc.CORINFO_HELP_CHKCASTANY:
+                case CorInfoHelpFunc.CORINFO_HELP_CHKCASTARRAY:
```

Comment:
You can delete `CORINFO_HELP_CHKCASTARRAY` and `CORINFO_HELP_ISINSTANCEOFARRAY` from JIT/EE interface. They are unnecessary now.